diff -dur cacti-plugin-npc-2.0.4.orig/npc/controllers/layout.php cacti-plugin-npc-2.0.4/npc/controllers/layout.php
--- cacti-plugin-npc-2.0.4.orig/npc/controllers/layout.php	2012-11-24 21:25:18.000000000 +0100
+++ cacti-plugin-npc-2.0.4/npc/controllers/layout.php	2012-11-24 21:26:15.649892901 +0100
@@ -58,10 +58,10 @@
             npc.params.npc_service_icons   = "<?php echo read_config_option('npc_service_icons'); ?>";
 
             npc.params.cacti_path          = "<?php echo URL_PATH; ?>";
-            var strLen = npc.params.cacti_path;
-            if (npc.params.cacti_path.charAt(strLen-1) == '/') {
-                npc.params.cacti_path = npc.params.cacti_path.slice(0,strLen-1);
-            }
+            //var strLen = npc.params.cacti_path;
+            //if (npc.params.cacti_path.charAt(strLen-1) == '/') {
+            //    npc.params.cacti_path = npc.params.cacti_path.slice(0,strLen-1);
+            //}
 
             <?php $state = unserialize(db_fetch_cell('SELECT settings FROM npc_settings WHERE user_id = ' . $_SESSION['sess_user_id'])); ?>
             var ExtState = Ext.decode('<?php echo json_encode($state); ?>');
